extensions/wikia/AdEngine/js/run/mercury.run.js

```diff
@@ -62,16 +75,15 @@ require([
 		actionHandler.registerMessageListener();
 	});
 
-	mercuryListener.onEveryPageChange(function () {
-		if (geo.isProperGeo(instantGlobals.wgAdDriverPrebidBidderCountries)) {
-			prebid.call();
-		}
-
-		if (
-			geo.isProperGeo(instantGlobals.wgAdDriverRubiconFastlaneCountries) &&
-			geo.isProperGeo(instantGlobals.wgAdDriverRubiconFastlaneMercuryFixCountries)
-		) {
-			rubiconFastlane.call();
-		}
-	});
+	// TODO: Remove else statement, this step is required in order to keep bidders working during cache invalidation
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)